site/content/en/docs/administration/advanced/k8s_deployment_with_helm.md

```diff
@@ -319,3 +326,26 @@ cvat:
             claimName: my-claim-name
 
 ```
+### Im receiving service unavailable errors
```

Comment:
Yeah, i aggree, the wording is vague, maybe something "Website not loading after running helm install, but redis opa and postgresql are running" ?

Reply:
I thinking more like:
```
### Im receiving service unavailable errors
There could be multiple explanations for that, but generally that is because your ingress controller do not recieve response from backend, some possible explanations are 
1. Server is down:
  A. your volume is RWO instead of RWX
  B. your DB is not responding
2. Service do not see pod:
  A. Check labels on pods
  B. Check label selector on service
3. 
####
```

Reply:
Good feedback, i improved it